src/radical/pilot/agent/scheduler/spark.py

```diff
@@ -0,0 +1,97 @@
+
+__copyright__ = "Copyright 2013-2016, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+import json
+import urllib2 as ul
+
+import radical.utils as ru
+
+from ... import utils     as rpu
+from ... import states    as rps
+from ... import constants as rpc
+
+from .base import AgentSchedulingComponent
+
+
+#===============================================================================
+#
+class Spark(AgentSchedulingComponent):
+
+    # FIXME: clarify what can be overloaded by Scheduler classes
+
```

Comment:
I think it was forgotten there.

See [here](https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/agent/scheduler/base.py#L27)